src/NServiceBus.Serverless.Tests/ApprovalFiles/APIApprovals.Approve.approved.txt

```diff
@@ -4,14 +4,13 @@ namespace NServiceBus.Serverless
     {
         protected ServerlessEndpoint(System.Func<TExecutionContext, NServiceBus.Serverless.ServerlessEndpointConfiguration> configurationFactory) { }
         public System.Threading.Tasks.Task Process(NServiceBus.Transport.MessageContext message, TExecutionContext executionContext) { }
+        public System.Threading.Tasks.Task ProcessFailedMessage(NServiceBus.Transport.ErrorContext errorContext, TExecutionContext executionContext) { }
     }
     public abstract class ServerlessEndpointConfiguration
     {
         protected ServerlessEndpointConfiguration(string endpointName) { }
         public NServiceBus.EndpointConfiguration AdvancedConfiguration { get; }
-        public void InMemoryRetries(int numberOfRetries) { }
-        public void SendFailedMessagesToErrorQueue(bool sendFailedMessagesToErrorQueue = True) { }
-        public void SendFailedMessageToErrorQueue(string errorQueueName) { }
+        public void SendFailedMessageToErrorQueue(string errorQueueName = "error") { }
```

Comment:
the current API would allow to explicitly disable sending messages to the error queue. Not sure this is really required but I'm not a huge fan of features which you can never turn off once you opted in at some point.

Reply:
I wonder if we even need this API given that we are going with a model where we require a transport